src/core/include/openvino/core/type/element_type.hpp

```diff
@@ -97,7 +97,6 @@ class OPENVINO_API Type {
     std::string get_type_name() const;
     friend OPENVINO_API std::ostream& operator<<(std::ostream&, const Type&);
 
-    OPENVINO_DEPRECATED("This function is deprecated and will be removed in 2026.0.")
```

Comment:
The function is not used in OV, different approach can be used.
If needed some developer API can be provided to have local helper which give compile time of supported list.

Reply:
I don't see anything wrong with leaving this function in api, other than this there is no way to get the list of supported types in openvino api. Please consider moving the function to dev api if the deprecation decision is already finalized. 
The current implementation of the validate method provides a good user experience, it is easy to align with the paged attention specification. All names, idx, types, mappings/rank checks are presented as a single line for each input.